src/components/user-research-banner/template.njk

```diff
@@ -18,9 +18,11 @@
     </div>
     <button class="govuk-link hmrc-user-research-banner__close">
       {% if params.language == 'cy' %}
-        Dim diolch
+        <span aria-hidden="true">Dim diolch</span>{#- strip spaces -#}
+        <span class="govuk-visually-hidden">Dim diolch, nid wyf am gymryd rhan yn yr ymchwil defnyddwyr - cuddio'r neges hon</span>
```

Comment:
I'm not sure how commas and hyphens work with welsh, or I guess specifically a welsh screen reader, guess it would be different pronunciation rules. I've sent a screenshot of the translation in place with hyphen visible for them to check if I did it correctly, I'll send another note to double check the hyphen is right

Reply:
@ellamdav they've come back and said it was deliberate, but that we could change it if needed back to a comma. I'm going to ask in int-plat-design if Shabani knows of any issues with hyphens and screen readers

Reply:
Shabana has said that we should go with the recommended translation, rather than trying to accomodate screen readers - so this change is ready now 👍